apple-pie/styles/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+    font-family: Arial;
+    font-size: 1em;
+    font-weight: lighter;
+    
+}
+
+.header {
+    
+    background-image: url("/apple-pie/images/apple-pie.jpg");
```

Comment:
` background-image: url("../images/apple-pie.jpg");` that way the image would be displayed